src/main/kotlin/app/revanced/patches/youtube/layout/returnyoutubedislike/fingerprints/TextComponentAtomicReferenceFingerprint.kt

```diff
@@ -0,0 +1,37 @@
+package app.revanced.patches.youtube.layout.returnyoutubedislike.fingerprints
```

Comment:
Sorry, I think I didn't clarify myself properly. You resolve one fingerprint on all classes/ methods. Later when you know the method, you can use the MethodFingerprint.resolve overload to resolve a method using another fingerprint. That other fingerprint does not need anything other than an opcode pattern if the fingerprint is only used to find the index of the matching pattern. That said, the first fingerprint is fine as is (looks like it could match another method accidentally because it doesn't look too unique) but the second fingerprint you need to resolve on the first fingerprints resulting method.

Reply:
Looks good, haven't seen some changes :)